app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesFragmentTest.kt

```diff
@@ -246,6 +276,58 @@ class PoliciesFragmentTest {
     }
   }
 
+  @Test
+  fun testPoliciesFragment_forTermsOfService_opensPrivacyPolicyPage() {
+    launch<PoliciesFragmentTestActivity>(
+      createPoliciesFragmentTestActivity(
+        getApplicationContext(),
+        PolicyPage.TERMS_OF_SERVICE
+      )
+    ).use { activityScenario ->
+      activityScenario.onActivity { activity ->
+
+        testCoroutineDispatchers.runCurrent()
+        val htmlParser = htmlParserFactory.create(
+          policyOppiaTagActionListener = mockPolicyOppiaTagActionListener,
+          displayLocale = appLanguageLocaleHandler.getDisplayLocale()
+        )
+        val textView: TextView =
+          activity.findViewById(R.id.policy_description_text_view)
+
+        val htmlResult: Spannable = htmlParser.parseOppiaHtml(
```

Comment:
Ditto here--this shouldn't be needed per the code being tested.

Reply:
removed.